src/spatialdata/_core/spatialdata.py

```diff
@@ -1574,6 +1574,50 @@ def subset(
         )
         return SpatialData(**elements_dict, tables=tables)
 
+    def validate_table_annotation_target(self, table_name: str) -> None:
```

Comment:
maybe check how much this compares to `validate_table_in_spatialdata`. Part of the functionality overlaps

Reply:
Thanks! I removed my function and put the extra functionalities inside yours.